index.js

```diff
@@ -1 +1,12 @@
-// code your solution here
+function superbowlWin(record) {
+  const gameResult = record.find(callback);
+  if (gameResult) {
+    return gameResult.year;
+  } else {
+    return undefined;
+  }
```

Comment:
Below will yield identical results with less code. 
```suggestion
  if (gameResult) {
    return gameResult.year;
  }
```
Why do you think that is?

Reply:
because with find it returns either a match or undefined by defualt if no match is found. The else return is just repetitive

Reply:
Exactly! You got it!